### DISCUSSION TITLE
Implement the Fluent Design Acrylic effect into Chatterino

### DISCUSSION THREAD

**Discussion by Void48** [Chatterino/chatterino2#3223] [Ideas]
I think this would definitely make Chatterino feel more modern and fresh if its possible ![FeelsOkayMan](https://user-images.githubusercontent.com/85196642/132110001-5c31d3c0-d56a-4dd5-b8ee-7d47b34729de.png)👍

Here are some links that should help with implementing this:  
https://github.com/Extrimis/Win32-Acrylic-Effect
https://stackoverflow.com/questions/57626018/how-to-get-a-blurred-translucent-qml-window-similar-to-fluent-design-guidelines
https://forum.qt.io/topic/51055/enableblurbehindwindow-not-working

**Comment by Mm2PL** [comment]
I can see this being an option however it needs #1618. I would accept a PR for this but it would be best if this wasn't a windows only thing but a crossplatform thing instead.
